### DISCUSSION TITLE
is there any way to not show the `v` before scoop's package version?

### DISCUSSION THREAD

**Discussion by ingenarel** [badges/shields#10590] [Q&A] [answered]
here's what i mean:
i've made a scoop bucket that's aur -git package inspired. the version of a package is the git commit hash
![image](https://github.com/user-attachments/assets/f5d4d1d1-871f-45ed-be6c-945155a99284)

now the thing is the version says 1769fa..

but on the readme, i can see a `v` before it.
![image](https://github.com/user-attachments/assets/410273ac-41dd-443f-8cb0-ddc75e262680)

is there any way to not do that?

**Comment by chris48s** [comment]
I think we should modify the `addv()` helper to detect when a version looks like a commit hash rather than a version number. We already omit the v prefix if the version looks like a date.

**Reply by chris48s** [reply]
moved to an issue https://github.com/badges/shields/issues/10591

**Comment by chris48s** [comment, marked as answer]
You haven't posted a link to your badge but https://github.com/badges/shields/pull/10597 has been merged and deployed which should fix this
